llvm/docs/DirectXUsage.rst

```diff
@@ -78,9 +79,11 @@ both ``DXBC`` and ``DXIL`` outputs, and the ultimate goal is to support both as
 code generation targets in LLVM, the LLVM codebase uses a more neutral name,
 ``DXContainer``.
 
-The ``DXContainer`` format is sparsely documented in the functional
+The ``DXcontainer`` format is sparsely documented in the functional
```

Comment:
Is the change to lowercase "c" here intentional?  It looks like it is capitalized everywhere else.